cmd/logger/logger.go

```diff
@@ -259,17 +264,36 @@ func getTrace(traceLevel int) []string {
 	return trace
 }
 
-// LogIf prints a detailed error message during
+// LogAlwaysIf prints a detailed error message during
 // the execution of the server.
-func LogIf(ctx context.Context, err error) {
-	if Disable {
+func LogAlwaysIf(ctx context.Context, err error) {
```

Comment:
@kannappanr what about doing this differently, 

we can add ignoredErrs parameter, like the following way
`func LogIf(ctx context.Context, err error, ignoredErrs ...error)`, that way it will print an error only when the former is not one of the passed ignored errors.

What do you think ?

Reply:
ping @kannappanr